### PR TITLE
Use UTC time zone explicitly for isEpoch test

### PR DESCRIPTION
**Pull request by utzcoz**
The time zone will be different if test runs on different time zone
machine. And the isEpoch test is for UTC time zone, so we should add it
explicitly for Time.

### CODE REVIEW

**Comment by utzcoz** [comment]
I run this test on my machine with time zone `Asia/Shanghai`, it always failed. With this change, it can pass.

**Comment by utzcoz** [comment]
Hi @hoisie , could you help to review this PR? Thanks.

**Comment by utzcoz** [comment]
I also found another problem of `isEpoch` test, the robolectric failed on negative test for `isEpoch` but it succeed on emulator:

```java
Time t = new Time("Asia/Shanghai");
assertThat(Time.isEpoch(t)).isFalse();
```

File: integration_tests/ctesque/src/test/java/android/text/format/TimeTest.java [hunk 1]
```diff
@@ -300,10 +297,9 @@ public void shouldFormat3339() throws Exception {
   }
 
   @Test
-  public void testIsEpoch() throws Exception {
-    Time t = new Time();
-    boolean isEpoch = Time.isEpoch(t);
-    assertEquals(true, isEpoch);
+  public void testIsEpoch() {
+    Time t = new Time("Etc/UTC");
```
Comment (hoisie) [inline integration_tests/ctesque/src/test/java/android/text/format/TimeTest.java:301]: nit: `Time time = new Time(Time.TIMEZONE_UTC);`

Comment (utzcoz) [reply integration_tests/ctesque/src/test/java/android/text/format/TimeTest.java]: Thanks. Done.

**Comment by hoisie** [comment]
Thanks!
